mlflow/utils/search_utils.py

```diff
@@ -272,7 +297,20 @@ def _does_run_match_clause(cls, run, sed):
                                   error_code=INVALID_PARAMETER_VALUE)
         if lhs is None:
             return False
-        if comparator in cls.filter_ops.keys():
+
+        if comparator in ['like', 'ilike']:
+            # Change value from sql syntax to regex syntax
+            if comparator == 'ilike':
+                value = value.lower()
+                lhs = lhs.lower()
+            if not value.startswith('%'):
+                value = '^' + value
+            if not value.endswith('%'):
+                value = value + '$'
+            value = value.replace('_', '.').replace('%', '.*')
```

Comment:
Not clear on the logic here--can you help understand this translation?
I tried sqlite and `%` should work.

Reply:
Same that the previous comment, it is only used by file store, so % (match any characters) is remplaced by .* in a regex.